blueocean-dashboard/src/main/js/components/parameter/Password.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { FormElement, PasswordInput } from '@jenkins-cd/design-language';
+import { propTypes } from './commonProptypes';
+// import { DebugRender } from './DebugRender';
+
+export class Password extends Component {
+
+    render() {
+        // const debugging = React.createElement(DebugRender, this.props);
+        // FIXME: defaultValue does not contain the value we wait for
```

Comment:
not likely an issue as discussed. WTF is there a default value for a password for ;)